### DISCUSSION TITLE
Recommendations: flexible reporting engine recommendations that can work well with snipeIT?

### DISCUSSION THREAD

**Discussion by tangeloservices** [snipe/snipe-it#13331] [Q&A]
Hey team,

We're using SnipeIT a good bit and want to use it a lot more for tracking both our and our clients' IT assets including software licenses. We're finding workarounds to get information out of snipeIT so far, but it would be really great to have more ability to create customised and calculated reports. I realise this is outside of snipeIT's scope so was hoping for recommendations from users with experience doing this kind of thing. 

What reporting tools might you recommend we investigate for backing up to snipeIT and potentially other data sources like our CRM to create and fill report templates? We're all reasonably capable of writing our own code, but would probably need a platform our sales and finance teams can use too. 

Thanks kindly

**Comment by TShiremanLCOG** [comment]
If you can use PowerBI that might be the easiest. Hook directly up to the DB and create your reports from there. Or, if you don't want the direct connection, you can likely make use of the generated backups and convert the SQL file to a CSV/Excel file and use those tools.
